src/main/java/marquez/api/CreateJobRunResponse.java

```diff
@@ -1,20 +1,17 @@
 package marquez.api;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import org.hibernate.validator.constraints.NotBlank;
 
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
 public final class CreateJobRunResponse {
-  private final UUID externalGuid;
-
-  @JsonCreator
-  public CreateJobRunResponse(@JsonProperty("run_id") @NotBlank final UUID externalGuid) {
-    this.externalGuid = externalGuid;
-  }
-
-  @JsonProperty("run_id")
-  public UUID getExternalGuid() {
-    return externalGuid;
-  }
+  @JsonProperty("runId")
+  @NotBlank
+  private UUID externalGuid;
```

Comment:
rename to `runId` locally too.

Reply:
Removed the file b/c we can use another api type to create the response. It's runId elsewhere.